### DISCUSSION TITLE
pyMOR Community Meeting 2023-10-18

### DISCUSSION THREAD

**Discussion by sdrave** [pymor/pymor#2196] [Announcements]
The next pyMOR Community Meeting will take place Wednesday, October 18, at 16:00 CEST and 10:00 EDT, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] Point release? (@sdrave)
- [x] Linting PRs (@pmli)
- [x] How-to guides (@pmli)
- [x] Matrix conversion #2201 (@artpelling)
- [x] QR algorithms in pyMOR (@sdrave)

Moderator: @sdrave

**Comment by lbalicki** [comment]
Meeting Notes:
- Point release
  - right now in the release documentation binder links do not work since they are on the old setup
  - 3.11 support
  - until end of this week include everything necessary for point release
  - do point release next week
- Linting PRs
  - we should take a look at toml PR first #2161 
  - concern is it adds an additional dependency which might break in the future
  - review PR and merge, we can remove it if it starts causing issues at some point
- How-to guides
  - idea: Tutorials are for beginners and how-to guides are for experts/users looking for something specific
- Matrix conversion
  - @artpelling will start PR on this
- QR algorithms in pyMOR
  - Good first step: Public interface function
  - many questions: Pivoting, tolerances, full vs rank-revealing,...
  - could have rrqr and qr
  - for now could be rrqr via gram_schmidt and qr via e.g. Cholesky-based algorithm

Next Meeting: Thursday, November 2nd, 14:00 CEST and 8:00 EDT
Moderator: @pmli
